docs/reference/data-streams/lifecycle/apis/put-global-retention.asciidoc

```diff
@@ -0,0 +1,131 @@
+[[data-streams-put-global-retention]]
+=== Updates the global retention of data streams
```

Comment:
I think this is supposed to be called something like `Update the global retention of data streams`. Most of our other pages like this start with imperative verbs (I think that's what it's called, I don't really know).